python/lsst/ts/ATDome/dome_csc.py

```diff
@@ -87,7 +92,10 @@ class ATDomeCsc(salobj.BaseCsc):
     * 1: could not connect to TCP/IP ATDome controller
     * 2: read from TCP/IP ATDome controller timed out
```

Comment:
LTS-441 specifies ranges for error codes. This is a topic that we still need to figure out how to manage (the ranges for each component). Current specification state that AT components error codes must be in the range 8201-8500.  Since we don't have a way to reserve ranges now, I would at least recommend to document the issue here and maybe add a `FIXME` statement...

Reply:
Created https://jira.lsstcorp.org/browse/DM-19530 and added a reference